packs/data/feat-effects.db/effect-pistoleros-challenge-legendary.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_id": "1q4VIzG7mtYykEsf",
+    "img": "systems/pf2e/icons/equipment/weapons/flintlock-pistol.webp",
+    "name": "Effect: Pistolero's Challenge (Legendary)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Pistolero's Challenge]{Pistolero's Challenge}</p>\n\n<p>Both you and the opponent gain a +2 status bonus to damage rolls with Strikes made against each other. If you're a master in the skill you used for the check, the status bonus to damage rolls increases to +3, and if you're legendary, it increases to +4.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "encounter",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "target:effect:pistoleros-challenge-legendary"
```

Comment:
This will need to be a toggle on the sheet, not something keyed off the effect being present. That's to prevent party's with more than one person with this feat. Honestly this is probably more a job for the eventually finished MarkTarget RE.

Reply:
Yeah, that's what I expected. I'll keep using it in our game, but if the other comment can't be resolved, it's probably best to postpone this.

Reply:
Gotcha. I'll get back to it once MarkTarget exists.